management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -329,8 +330,7 @@ public Response reloadLocalSchema() {
       content =
           @Content(
               mediaType = MediaType.APPLICATION_JSON,
-              schema = @Schema(implementation = String.class),
-              examples = @ExampleObject(value = STREAMING_INFO_RESPONSE_EXAMPLE)))
+              schema = @Schema(implementation = StreamingInfo.class)))
```

Comment:
Suggestion: Is `STREAMING_INFO_RESPONSE_EXAMPLE` still used anywhere? Can we remove that ugly block of text?

Reply:
I think I did.